### PR TITLE
Add COMPOSITION in KeyPath to target root composition layer.

### PR DESCRIPTION
**Pull request by boduan-coder**

### CODE REVIEW

**Comment by gpeal** [comment]
@boduan-coder Can you add some javadocs? Other than that, lgtm!

File: issue-repro/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/Theme.AppCompat.Light.NoActionBar">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MyActivity">
```
[the remark this answers is not in the document]
Comment (windkiosk) [reply issue-repro/src/main/AndroidManifest.xml:9]: Oh, yeah, I will revert the changes for test purpose.

[the remark this answers is not in the document]
Comment (boduan-coder) [reply issue-repro/src/main/AndroidManifest.xml]: Sorry, I have reverted the code for test purpose.

**Comment by boduan-coder** [comment]
@gpeal , is the screenshot test automatically triggered? I can't see the test report.

**Comment by gpeal** [comment]
@boduan-coder I accidentally broke them but just fixed them here https://github.com/airbnb/lottie-android/pull/1563

Could you merge then push to kick it off again?

File: lottie/src/main/java/com/airbnb/lottie/model/KeyPath.java [hunk 2]
```diff
@@ -35,12 +36,17 @@
...
+ *     Target the the root composition layer:
+ *        KeyPath.COMPOSITION
  *
  *
  * NOTE: Content that are part of merge paths or repeaters cannot currently be resolved with
  * a {@link KeyPath}. This may be fixed in the future.
  */
 public class KeyPath {
+  // A singleton KeyPath that targets on the root composition layer.
+  // This is useful if you want to apply transformer to the animation as a whole.
```
Comment (gpeal) [inline lottie/src/main/java/com/airbnb/lottie/model/KeyPath.java:48]: nit: use the class javadoc style so you get the right documentation popup in Android Studio

Comment (boduan-coder) [reply lottie/src/main/java/com/airbnb/lottie/model/KeyPath.java]: Good catch. I thought they were same in term of inline comments. Just realized the difference.

**Comment by boduan-coder** [comment]
@gpeal , could you please help merge this PL if everything looks good?

**Comment by boduan-coder** [comment]
Friendly ping :)

**Pull request merged**